src/System.Net.Security/src/System/Net/NTAuthentication.cs

```diff
@@ -429,13 +341,11 @@ internal byte[] GetOutgoingBlob(byte[] incomingBlob, bool throwOnError, out Inte
                 if (!_isServer)
                 {
                     // client session
-                    statusCode = (Interop.SecurityStatus)SSPIWrapper.InitializeSecurityContext(
-                        GlobalSSPI.SSPIAuth,
+                    statusCode = NegoState.InitializeSecurityContext(
```

Comment:
Why NegoState depends on NTAuthentication as a context, and NTAuthentication also reference back to NegoState ?